tensorflow/core/kernels/random_op_gpu.cu.cc

```diff
@@ -223,9 +223,9 @@ void FillPhiloxRandom<GPUDevice, Distribution>::operator()(
       (d.getNumCudaMultiProcessors() * d.maxCudaThreadsPerMultiProcessor()) /
```

Comment:
definitely could be merged together now as APIs in Eigen have be consolidated.

Reply:
you are right...will combine the two